config/default/StuckTasksCheck.conf.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+use oat\taoSystemStatus\model\Check\System\StuckTasksCheckService;
+
+return new \oat\taoSystemStatus\model\Check\System\StuckTasksCheckService([
+    StuckTasksCheckService::RUNNING_MAX_TIME => 2, // max time for running status, hours
```

Comment:
let's update it to minutes. Sometimes we would need to check less than 1h

Reply:
fixed